j/javax-inject/javax-inject_ubuntu_20.04.sh

```diff
@@ -0,0 +1,31 @@
+# ----------------------------------------------------------------------------
+#
+# Package	: javax-inject
+# Version	: 4.0.0 
+# Source repo	: https://github.com/javax-inject/javax-inject 
+# Tested on	: ubuntu_20.04
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Gururaj R Katti <Gururaj.Katti@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+sudo apt-get update
+sudo apt-get install openjdk-8-jdk wget git -y
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-ppc64el/
+
+wget http://www-eu.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
+tar xzvf apache-maven-3.6.3-bin.tar.gz
+export PATH=$PATH:`pwd`/apache-maven-3.6.3/bin
+git clone https://github.com/javax-inject/javax-inject
```

Comment:
I believe the reason we are going with top-of-tree/master is that this project does not have a current release or tag that we can use, is that so?

Reply:
Yes @seth-priya it doesn't have any release tags